BudgetSquirrel.Api/ClientApp/src/app/splash-page/splash-page.component.scss

```diff
@@ -0,0 +1,98 @@
+@import './../../variables';
```

Comment:
Good question. Nick has a whole set of designs for mobile. So yeah we have one but I think that should be a stretch goal for now, especially since this is the 3rd time we restarted this app lol.